packages/x-data-grid-premium/src/tests/dataSourceAggregation.DataGridPremium.test.tsx

```diff
@@ -104,6 +104,8 @@ describeSkipIf(isJSDOM)('<DataGridPremium /> - Data source aggregation', () => {
       expect(fetchRowsSpy.callCount).to.be.greaterThan(0);
     });
     await user.click(within(getColumnHeaderCell(0)).getByLabelText('id column menu'));
+    // wait for the column menu to be open first
+    await screen.findByRole('menu', { name: 'id column menu' });
```

Comment:
What a nasty test to fix 😅
Do you think this will make any difference?

Reply:
I'm no longer sure... 🤷 
It's hard to debug it when locally it's impossible to get a failure. 🙈 

But at least on Pickers, I often tend to first assert that a Popper has been attached and only then do the actual assertion. 🤔

Reply:
The popper tests are extremely frustrating to test. There is no exact reason why they fail as far as I could tell

Reply:
And it failed once again... 🤷